cmd/tracee-ebpf/main.go

```diff
@@ -313,7 +315,7 @@ func main() {
 			&cli.BoolFlag{
 				Name:        "debug",
 				Value:       false,
-				Usage:       "write verbose debug messages to standard output and retain intermediate artifacts",
+				Usage:       "write verbose debug messages to standard output and retain intermediate artifacts. this may break the event stream if --output:out-file is set to stdin, so a different file should be provided",
```

Comment:
I think you meant stdout, but anyway I don't think this is accurate.. the promise for the output of tracee-ebpf to be just events holds regardless if the output is sent to stdout or to a file. I would suggest: `enabling debug printing will pollute the output with informative messages which will likely to break consumers that expected the output to be machine-readable event stream`

Reply:
Your version misses the fact that this is only relevant if both events and debug messages go to stdout (if the streams are separate there will be no problem), but it does reflect the message more clearly, So I've edited to: `enabling will output debug messages to stdout, which will likely break consumers which expect to receive machine-readable events from stdout` WDYT?

Reply:
Ah I see now what you meant initially. I didn't realize we were println-ing straight away. This wording sgtm